test/integration/build_to_buildruns_test.go

```diff
@@ -232,10 +233,10 @@ var _ = Describe("Integration tests Build and BuildRuns", func() {
 
 			br, err := tb.GetBR(buildRunObject.Name)
 			Expect(err).To(BeNil())
-			Expect(br.Status.Reason).To(Equal(fmt.Sprintf("Build.shipwright.io \"%s\" not found", BUILD+tb.Namespace)))
+			Expect(br.Status.Reason).To(Equal(fmt.Sprintf("build.shipwright.io \"%s\" not found", BUILD+tb.Namespace)))
 			Expect(br.Status.StartTime).To(BeNil())
 			Expect(br.Status.GetCondition(v1alpha1.Succeeded).Status).To(Equal(corev1.ConditionFalse))
-			Expect(br.Status.GetCondition(v1alpha1.Succeeded).Reason).To(Equal("Failed"))
+			Expect(br.Status.GetCondition(v1alpha1.Succeeded).Reason).To(Equal(resources.ConditionBuildNotFound))
```

Comment:
You do not use the constant in line 266 and 299. Is controversial sometimes because if the constant is wrong, then the wrong value is also asserted in the test case. So, just to make you aware and you may decide how to do it = I would also be fine if the constant is used at all places.

```suggestion
			Expect(br.Status.GetCondition(v1alpha1.Succeeded).Reason).To(Equal("BuildNotFound"))
```

Reply:
+1